salt/loader.py

```diff
@@ -1535,7 +1535,7 @@ def _load_module(self, name):
                         # pylint: enable=no-member
                         sys.modules[mod_namespace] = mod
                     else:
-                        with salt.utils.files.fopen(fpath, desc[1]) as fn_:
+                        with salt.utils.files.fopen(fpath, desc[1], use_io_open=False) as fn_:
```

Comment:
Eeek. Why are we modifying the loader in an attempt to fix a failing state test?

Reply:
The loader needs to open using the old method (not io.open).

Reply:
OK, but why? This is a change to the core of how Salt operates. We need to understand exactly what's happening here.

Reply:
@terminalmage can you answer this?

Reply:
Nevermind... this has already been fixed